### DISCUSSION TITLE
Can't seem to find FormProgressIndicator(parentContainer)

### DISCUSSION THREAD

**Discussion by Zackdazaa** [KhamisiKibet/QT-PyQt-PySide-Custom-Widgets#74] [Q&A]
Hi! I just found your project which is really helpful for me. However when I follow your guide on how to create custom progressbarindicator with steps I can't seem to fine this line of code in any file. It is use to create progressbarindicator the line is "myProgressBar = FormProgressIndicator(parentContainer)". So where can I find this line of code? it is not define by any libary or classes.
  
p.s. I've tried using QCustomProgressIndicator(QWidget) instead but it is not working either.
thankyou in advance.

**Comment by KhamisiKibet** [comment]
Download the examples from examples folder

**Reply by Zackdazaa** [reply]
I did. But in the example, there's no line called 'myProgressBar = FormProgressIndicator(parentContainer)' which use to create the progress bar. Instead, there's straight up this line 'self.ui.widget.updateFormProgressIndicator' which cause me a little confusion. I mean I am trying to create a custom progressbar without qt designer.
